flink-runtime/src/main/scala/org/apache/flink/runtime/taskmanager/TaskManager.scala

```diff
@@ -613,12 +625,11 @@ class TaskManager(
             log.debug("Ignoring duplicate registration acknowledgement.")
           } else {
             log.warn(s"Received 'AlreadyRegistered' message from " +
-              s"JobManager ${jobManager.path}, even through TaskManager is currently " +
+              s"ResourceManager ${jobManager.path}, even through TaskManager is currently " +
```

Comment:
This is an artifact of refactoring the original approach that I followed where the TaskManager would register at the ResourceManager. Reverting.